runner/runners/local_output.go

```diff
@@ -44,6 +44,10 @@ func NewHttpOutputCreator(tmp *temp.TempDir, httpUri string) (HttpOutputCreator,
 		}
 		httpPath = strings.TrimSuffix(u.Path, "/") + "/"
 	}
+	tmp, err := ioutil.TempDir("", "output")
```

Comment:
with the orig code, the output dir would be under scoot-tmp-<rand #>/ .... should we try to preserve that behavior?

Reply:
imo it doesn't really matter. we do still set TMPDIR=$(pwd)/scoot-tmp in our aurora file, so this output dir will be located at <path to aurora sandbox>/scoot-tmp/output-<rand #>